package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "commitplease",
-  "version": "2.3.1",
+  "version": "2.4.0",
```

Comment:
This bump is here because I rebased this branch (that was here since April) off the master. And the master got those important changes with ticketReferences. So, that is not a "manual" change, that is from master.